app/views/BookDetail/index.tsx

```diff
@@ -160,60 +196,56 @@ function BookDetail(props: BookProps) {
                         </Tab>
                         <Tab
-                            name="new-tab"
+                            name="content-tab"
                         >
                             <React.Fragment key=".0">
                                 Content
                             </React.Fragment>
                         </Tab>
-                        <Card>
-                            Lorem ipsum dolor sit amet, consectetur adipiscing
-                            elit. Nulla sed convallis quam, quis molestie nisi.
-                            Integer fringilla maximus tellus at aliquam.
-                            Nunc ac turpis non elit placerat luctus. Mauris vehicula,
-                            dui vitae feugiat malesuada, diam elit porttitor
-                            tellus, ut ultricies nibh est at ante.
-                            Maecenas congue congue nulla quis feugiat.
-                            Etiam porta volutpat mollis. Morbi libero eros,
-                            malesuada nec metus ac, varius cursus purus.
-                            Proin metus tellus, fermentum vel tellus et,
-                            tristique mattis urna. Nunc sapien sapien, malesuada
-                            posuere nulla in, imperdiet placerat o
-                            Phasellus dapibus magna sit amet neque sollicitudin
-                            laoreet.
-                        </Card>
+                        <TabPanel name="description-tab">
+                            {book.description}
+                        </TabPanel>
                     </Tabs>
                 </Container>
                 <Container
                     className={styles.authorDescription}
                     heading="About the Author"
                 />
-                <Card>
-                    Lorem ipsum dolor sit amet, consectetur adipiscing
-                    elit. Nulla sed convallis quam, quis molestie nisi.
-                    Integer fringilla maximus tellus at aliquam.
-                    Nunc ac turpis non elit placerat luctus. Mauris vehicula,
-                    dui vitae feugiat malesuada, diam elit porttitor
-                    tellus, ut ultricies nibh est at ante.
-                    Maecenas congue congue nulla quis feugiat.
-                    Etiam porta volutpat mollis. Morbi libero eros,
-                    malesuada nec metus ac, varius cursus purus.
-                    Proin metus tellus, fermentum vel tellus et,
-                    tristique mattis urna. Nunc sapien sapien, malesuada
-                    posuere nulla in, imperdiet placerat o
-                    Phasellus dapibus magna sit amet neque sollicitudin
-                    laoreet.
-                </Card>
+                {!loading && (
+                    <Card>
+                        {book.authors.forEach((author: any) => {
+                            <p>{author.name}</p>;
+                        })}
```

Comment:
1. Please use Array.map here. Array.forEach does not return a value.
2. We need to add `key` on each element.
3. We should not use `any` type

Reply:
Okay thank you